omnibus/resources/agent/msi/cal/stopservices.cpp

```diff
@@ -319,7 +319,7 @@ BOOL StopDependentServices(SC_HANDLE hScManager, SC_HANDLE hService)
 // Return value:
 //   None
 //
-DWORD __stdcall DoStartSvc(MSIHANDLE hInstall, std::wstring& svcname)
+DWORD  DoStartSvc(MSIHANDLE hInstall, std::wstring& svcname)
```

Comment:
Why remove `__stdcall` ?

Reply:
it was causing a 32-bit compile error (don't remember which one), and is unnecessary.  We only have to explicitly declare the calling convention on exported functions.